core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -464,7 +464,7 @@ private[spark] class Executor(
         executorSource.METRIC_OUTPUT_BYTES_WRITTEN
           .inc(task.metrics.outputMetrics.bytesWritten)
         executorSource.METRIC_OUTPUT_RECORDS_WRITTEN
-          .inc(task.metrics.inputMetrics.recordsRead)
+          .inc(task.metrics.outputMetrics.recordsWritten)
```

Comment:
Could you remove `[MINOR]`? Actually, SPARK-25536 issue is marked as 'MAJOR' bug.

Reply:
In addition, please add more description about when this bug is introduced. e.g. `SPARK-22190` in this case.

Reply:
Thanks @dongjoon-hyun . I have updated the PR description.